_drafts/PR-blogs/planning-docs/example-code/tic-tac-toe/original/script.js

```diff
@@ -0,0 +1,87 @@
+const statusDisplay = document.querySelector('.game--status');
+
+let gameActive = true;
+let currentPlayer = "X";
+let gameState = ["", "", "", "", "", "", "", "", ""];
+
+const winningMessage = () => `Player ${currentPlayer} has won!`;
```

Comment:
Excellent practice to wrap things like this up into functions instead of hardcoded strings.